src/Data/ArrayBuffer/ArrayBuffer.js

```diff
@@ -6,31 +6,41 @@ exports.create = function(s) {
   return function () {
     return new ArrayBuffer(s);
   };
-}
+};
 
 exports.byteLength = function(a) {
   return a.byteLength;
-}
+};
 
 exports.sliceImpl = function(s, e, a) {
   return function () {
     return a.slice(s, e);
   };
-}
+};
 
 exports.fromArray = function(s) {
   return (new Uint8Array(s)).buffer;
-}
+};
 
 exports.fromIntArray = function(s) {
   return (new Uint8Array(s)).buffer;
-}
+};
 
 exports.fromString = function(s) {
```

Comment:
Was there any issue in the previous implementation besides the missing semicolon?

Reply:
Nah, I was just playing cleanup while I was in there.